lib/airtime.js

```diff
@@ -49,22 +49,66 @@ class Airtime {
         });        
     }
 
-    validateOptions(options) {
-        const schema = Joi.object().keys({
-            idempotencyKey: Joi.string().optional(),
-            recipients: Joi.array().items(
-                Joi.object({
-                    phoneNumber:Joi.string().required().custom((value)=>{
-                        return phoneValidator(value).isValid? value : new Error(`the phone number ${value} is invalid`);
+     findTransactionStatus(options){
```

Comment:
@ibrixze why not take `transactionId` as the only param to make validation easier? i.e. no need to do `context:...` on validation

Reply:
Okay, I will do it